.github/workflows/publish-java-apidocs.yml

```diff
@@ -22,22 +31,15 @@ jobs:
           build-root-directory: java
           gradle-executable: java/gradlew
           arguments: javadoc
-      - name: Set commit ID
+      - name: Get source commit
         id: vars
         run: echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
-      - uses: actions/checkout@v2
-        with:
-          ref: gh-pages
-          clean: false
-      - name: Move API docs into target area
-        run: |
-          rm -rf docs/api/java
-          mv java/build/docs/javadoc docs/api/java
-      - name: Create Pull Request
-        uses: peter-evans/create-pull-request@v3
+      - name: Log source commit
+        run: echo $(git rev-parse --short HEAD) > java/build/docs/javadoc/version.txt
```

Comment:
- is the version.txt content visible somewhere in the generated docs?
- nit: the name version.txt may be ambiguous, i.e., it could be confused with the ORT version
- nit: `git rev-parse --short HEAD > java/build/docs/javadoc/version.txt` should work directly

Reply:
Thanks @edgchen1 - I can make these changes in the subsequent PR for the other docs